### ISSUE TITLE
Model Tuning

### ISSUE THREAD

**Issue by monkey1208** [lukemelas/image-paragraph-captioning#12]
Thanks for sharing all the work you done!

I used bottom-up-attention model given by Peteanderson80 to extract image features because I need features more than parabu_att such as boxes' coordinates.

But I found that using the parabu_att extracted from bottom-up-attention only gives CIDEr 0.25. 

I believed the hyperparameters provided in your code must be discovered after hundreds of tuning .
So I'm curious about how and what you work on tuning the model.
Thanks!!!

**Comment by monkey1208** [comment]
Oh, I just try adjusting learning rate and got CIDEr 0.3, I'll close this issue.

**Comment by RunCode90** [comment]
I'm glad to hear that you have done this image paragraph task. I am also interesting in the image paragraph,  but I don't know how to obtain the pretrained features: parabu_fc and parabu_att.
Can you share a link about the pretraining features (parabu_fc and parabu_att.) you have obtained?

Thanks a lot. 
Best wishes.

**Comment by RunCode90** [comment]
@monkey1208
I'm glad to hear that you have done this image paragraph task. I am also interesting in the image paragraph, but I don't know how to obtain the pretrained features: parabu_fc and parabu_att.
Can you share a link about the pretraining features (parabu_fc and parabu_att.) you have obtained?

Thanks a lot.
Best wishes.
